torch/csrc/lazy/core/tensor_impl.h

```diff
@@ -36,6 +37,8 @@ class TORCH_API LTCTensorImpl final : public c10::TensorImpl {
 
   int64_t stride(int64_t d) const override;
 
+  virtual c10::SymIntArrayRef sym_sizes() const;
```

Comment:
With the latest design, this doesn't actually need to be virtual; also, it should be on TensorImpl.h itself

Reply:
that seems correct, yeah